get-thin-edge_io.sh

```diff
@@ -134,11 +134,11 @@ esac
 rm -R /tmp/tedge
 
 # Test if tedge command is there and working
-tedge help >/dev/null
-if [ $? -eq 0 ]; then
-    echo "\n${BLUE}thin-edge.io is now installed on your system!${COLORRESET}"
+if tedge help >/dev/null;
+then
+    echo -e "\n${BLUE}thin-edge.io is now installed on your system!${COLORRESET}"
```

Comment:
In order to be `sh` compatible.

```suggestion
    echo
    echo "${BLUE}thin-edge.io is now installed on your system!${COLORRESET}"
```